app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -47,61 +62,145 @@ const useStyles = makeStyles((theme) => ({
     height: "50vh",
     justifyContent: "center",
     textAlign: "center",
+    [theme.breakpoints.up("md")]: {
+      height: "100%",
+      margin: "0 auto",
+      width: "960px",
+    },
+  },
+  desktopNavigation: {
+    display: "flex",
+  },
+  introduction: {
+    color: "#ffffff",
+    textAlign: "left",
+    width: "742px",
+  },
+  loginLink: {
+    border: `1px solid ${theme.palette.primary.main}`,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    marginRight: theme.spacing(3),
+    padding: theme.spacing(1, 0),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
   },
-  navigation: {
+  mobileNavigation: {
     display: "flex",
     justifyContent: "space-around",
-    marginTop: theme.spacing(3),
+    marginTop: theme.spacing(3, 0),
+  },
+  signupLink: {
+    backgroundColor: theme.palette.primary.main,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    padding: theme.spacing(1),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
   },
   subtitle: {
     marginTop: theme.spacing(1),
+    [theme.breakpoints.up("md")]: {
+      display: "inline-block",
+      marginTop: theme.spacing(5),
+      position: "relative",
+    },
+  },
+  title: {
+    ...theme.typography.h1,
+    [theme.breakpoints.up("md")]: {
+      fontSize: "64px",
+      lineHeight: "74px",
```

Comment:
use no unit (which is a multiple of fontSize, so here it would be 1.153. 1.15 is fine)